sympy/combinatorics/fp_groups.py

```diff
@@ -494,6 +496,120 @@ def elements(self):
         P, T = self._to_perm_group()
         return T.invert(P._elements)
 
+    @property
+    def is_polycyclic(self):
+        if self.is_solvable:
+            return True
+        return None
+
+    def is_subgroup(self, other):
+        """
+        Return ``True`` if all elements of ``self`` belong to ``other``.
```

Comment:
This is a misleading docstring, because for two `FpGroup`s on differently-sized `FreeGroup`s this is never true. A more correct statement would be "Check if `self` contains an isormorphic copy of `other`". But the method you are using won't necessarily be able to determine that. `other._to_perm_group()` could theoretically return a group of permutations that are not part of `self._to_perm_group()`